analyses/org.osate.analysis.resource.budgets/model/busload.genmodel

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<genmodel:GenModel xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore"
+    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" modelDirectory="/org.osate.analysis.resource.budgets/src-gen" editDirectory=""
+    copyrightText="Copyright (c) 2004-2021 Carnegie Mellon University and others. (see Contributors file).&#xA;All Rights Reserved.&#xA;&#xA;NO WARRANTY. ALL MATERIAL IS FURNISHED ON AN &quot;AS-IS&quot; BASIS. CARNEGIE MELLON UNIVERSITY MAKES NO WARRANTIES OF ANY&#xA;KIND, EITHER EXPRESSED OR IMPLIED, AS TO ANY MATTER INCLUDING, BUT NOT LIMITED TO, WARRANTY OF FITNESS FOR PURPOSE&#xA;OR MERCHANTABILITY, EXCLUSIVITY, OR RESULTS OBTAINED FROM USE OF THE MATERIAL. CARNEGIE MELLON UNIVERSITY DOES NOT&#xA;MAKE ANY WARRANTY OF ANY KIND WITH RESPECT TO FREEDOM FROM PATENT, TRADEMARK, OR COPYRIGHT INFRINGEMENT.&#xA;&#xA;This program and the accompanying materials are made available under the terms of the Eclipse Public License 2.0&#xA;which is available at https://www.eclipse.org/legal/epl-2.0/&#xA;SPDX-License-Identifier: EPL-2.0&#xA;&#xA;Created, in part, with funding and support from the United States Government. (see Acknowledgments file).&#xA;&#xA;This program includes and/or can make use of certain third party source code, object code, documentation and other&#xA;files (&quot;Third Party Software&quot;). The Third Party Software that is used by this program is dependent upon your system&#xA;configuration. By using this program, You agree to comply with any and all relevant Third Party Software terms and&#xA;conditions contained in any such Third Party Software or separate license file distributed with such Third Party&#xA;Software. The parties who own the Third Party Software (&quot;Third Party Licensors&quot;) are intended third party beneficiaries&#xA;to this license with respect to the terms applicable to their Third Party Software. Third Party Software licenses&#xA;only apply to the Third Party Software and not any other portion of this program or this program as a whole.&#xA;"
+    editorDirectory="" modelPluginID="org.osate.analysis.resource.budgets" modelName="Busload"
+    updateClasspath="false" rootExtendsClass="org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container"
+    testsDirectory="" importerID="org.eclipse.emf.importer.ecore" complianceLevel="8.0"
+    copyrightFields="false" usedGenPackages="../../org.osate.analysis.model/model/analysis.genmodel#//analysis ../../org.osate.aadl2/model/instance.genmodel#//instance ../../org.osate.aadl2/model/aadl2.genmodel#//aadl2"
+    operationReflection="true" importOrganizing="true">
+  <foreignModel>busload.ecore</foreignModel>
+  <genPackages prefix="Busload" basePackage="org.osate.analysis.resources.budgets.internal.models"
```

Comment:
should be "resource" instead of "resources" in the base package

Reply:
done